### PR TITLE
Refactor fmt for log

### PR DESCRIPTION
**Pull request by phillbaker**
Swaps for for fatalf, fatalln, printf, etc.

cc @aybabtme @bryanl @macb

### CODE REVIEW

File: account.go [hunk 1]
```diff
@@ -33,11 +33,10 @@ func accountShow(ctx *cli.Context) {
 	client := godo.NewClient(oauthClient)
 
 	account, _, err := client.Account.Get()
-
 	if err != nil {
-		fmt.Printf("%s\n", err)
-		os.Exit(1)
+		log.Fatalln(err)
```
Comment (aybabtme) [inline account.go]: you don't need the `ln` in `Fatalln`, newlines are inserted automatically

File: account.go [hunk 2]
```diff
@@ -33,11 +33,10 @@ func accountShow(ctx *cli.Context) {
...
 	}
+
 	fmt.Printf(
```
Comment (aybabtme) [inline account.go]: this should be to stderr

File: action.go [hunk 3]
```diff
@@ -38,8 +38,7 @@ var ActionCommand = cli.Command{
 
 func actionShow(ctx *cli.Context) {
 	if len(ctx.Args()) == 0 || len(ctx.Args()) > 1 {
-		fmt.Printf("Error: Must provide exactly one id of an Action to show.\n")
-		os.Exit(64)
+		log.Fatalln("Error: Must provide exactly one id of an Action to show.")
```
Comment (aybabtme) [inline action.go]: same here, don't use `ln`

File: action.go [hunk 4]
```diff
@@ -79,8 +77,7 @@ func actionList(ctx *cli.Context) {
 	actionList, _, err := client.Actions.List(opt)
 
 	if err != nil {
-		fmt.Printf("Unable to list Actions: %s\n", err)
-		os.Exit(1)
+		log.Fatalf("Unable to list Actions: %s\n", err)
```
Comment (aybabtme) [inline action.go]: don't need the `\n`

File: action.go [hunk 5]
```diff
@@ -53,8 +52,7 @@ func actionShow(ctx *cli.Context) {
 	action, _, err := client.Actions.Get(int(id))
 
 	if err != nil {
-		fmt.Printf("%s\n", err)
-		os.Exit(1)
+		log.Fatalln(err)
```
Comment (aybabtme) [inline action.go]: same

**Comment by aybabtme** [comment]
Everywhere:

- [ ] replace the calls to `Funcln` by `Func`
- [ ] remove the trailing `\n` from format strings passed to `log.Funcf`
- [ ] remove all calls to `fmt.Func`
- [ ] carefully define what the output of `stdout` should be for calls, if any. it should be somewhat machine readable and not have ASCII drawings

**Comment by phillbaker** [comment]
Updated.

File: droplet.go [hunk 6]
```diff
@@ -156,7 +156,7 @@ func dropletCreate(ctx *cli.Context) {
 			if sshKey != nil && err == nil {
 				sshKeys = append(sshKeys, godo.DropletCreateSSHKey{ID: sshKey.ID})
 			} else {
-				fmt.Printf("Warning: Could not find key: %s.\n", keyName)
+				log.Fatalf("Warning: Could not find key: %s", keyName)
```
Comment (aybabtme) [inline droplet.go]: use `%q` instead of `%s` to print keys, so that empty strings are quotted

File: droplet.go [hunk 7]
```diff
@@ -402,8 +383,7 @@ func dropletActionPowercycle(ctx *cli.Context) {
 
 func dropletActionShutdown(ctx *cli.Context) {
 	if ctx.Int("id") == 0 && len(ctx.Args()) != 1 {
-		fmt.Printf("Error: Must provide ID or name for Droplet to destroy.\n")
-		os.Exit(1)
+		log.Fatal("Error: Must provide ID or name for Droplet to issue shutdown.")
```
Comment (aybabtme) [inline droplet.go]: some messages are punctuated (dot), some aren't. would be good to have them all punctuated

**Comment by aybabtme** [comment]
lgtm minus the two nitpicks

File: main_test.go [hunk 8]
```diff
@@ -54,8 +54,8 @@ func TestGlobalFormatFlag(t *testing.T) {
 	args := []string{"doctl", "-k", "key", "-f", "invalid"}
 	err := app.Run(args)
 
-	expected := "invalid output format: invalid, available output options: json, yaml"
+	expected := "invalid output format: \"invalid\", available output options: json, yaml."
```
Comment (aybabtme) [inline main_test.go]: raw strings:
```go
`invalid output format: "invalid", available output options: json, yaml.`
```

**Comment by aybabtme** [comment]
:shipit:

**Pull request merged**